src/actions/actionTypes.js

```diff
@@ -6,6 +6,9 @@ export const PASSWORD_RESET= "PASSWORD_RESET";
 export const NEW_PASSWORD="NEW_PASSWORD";
 export const USER_NOT_FOUND="USER_NOT_FOUND";
 export const REGISTER_USER= "REGISTER_USER";
+export const RATE_ARTICLE="RATE_ARTICLE";
+export const CHECK_OWNER="CHECK_OWNER";
+export const FINISHED_RATING="FINISHED_RATING";
 const ACTION_TYPE = {
   NEW_ARTICLE: "NEW_ARTICLE",
   VIEW_ARTICLES: "VIEW_ARTICLES",
```

Comment:
Add your actions to the ACTION_TYPE dictionary, and use the exported ACTION_TYPE to call your actions, so as to have code consistency